hack/get-build.sh

```diff
@@ -18,53 +18,59 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source "${KUBE_ROOT}/cluster/common.sh"
+
 declare -r KUBE_RELEASE_BUCKET_URL="https://storage.googleapis.com/kubernetes-release"
 declare -r KUBE_TAR_NAME="kubernetes.tar.gz"
 
 usage() {
-	echo "usage:
-  $0 [stable|release|latest|latest-green]
-
-        stable:        latest stable version
-        release:       latest release candidate
-        latest:        latest ci build
-        latest-green:  latest ci build to pass gce e2e"
```

Comment:
If people are scripting off of this, their scripts will definitely break.  I thought that was a lesser evil than keeping these aliases around, but I'm definitely willing to be overruled.